workorders.lic

```diff
@@ -76,9 +71,29 @@ class WorkOrders
         shape_items(info, item, quantity)
       end
     when 'carving'
+      @material_type = @use_bone ? 'bone' : 'stone'
       @engineering_room = settings.engineering_room
       tools = settings.carving_tools
       @belt = settings.engineering_belt
+      
+      if !@use_bone 
```

Comment:
Thank you, I think I understand the *logical* difference in the two blocks, but I can't see any *code* difference. Could you help me understand how the code differs?